lib/utils/fs.go

```diff
@@ -209,3 +211,42 @@ func FSTryReadLockTimeout(ctx context.Context, filePath string, timeout time.Dur
 
 	return unlockWrapper(fileLock.Unlock, fileLock.Path()), nil
 }
+
+// RemoveSecure attempts to securely delete the file by first overwriting the file with random data followed by
+// calling os.Remove(filePath). The number of times to overwrite the data is set by iterations. Iterations must
+// be > 0 or a trace.BadParameterError will be returned.
+func RemoveSecure(filePath string, iterations int) error {
+	if iterations < 1 {
+		return trace.BadParameter("iterations must be greater than 0")
+	}
+
+	for i := 0; i < iterations; i++ {
+		if err := overwriteFile(filePath); err != nil {
+			return trace.Wrap(err)
+		}
+	}
+	return trace.ConvertSystemError(os.Remove(filePath))
+}
+
+func overwriteFile(filePath string) error {
+	f, err := os.OpenFile(filePath, os.O_WRONLY, 0)
+	if err != nil {
+		return trace.ConvertSystemError(err)
+	}
+	defer f.Close()
+
+	fi, err := f.Stat()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	// Rounding up to 4k to hide the original file size. 4k was chosen because it's a common block size.
+	var block int64 = 4096
```

Comment:
```suggestion
	const block = 4096
```

It looks to me that dropping the type would work, but if not just add it back.

Reply:
Thanks @codingllama, you are correct. I made the change.